src/main/java/ovh/knowall/socialstats/api/twitter/ShowMeWhatYouGotApplication.java

```diff
@@ -0,0 +1,13 @@
+package ovh.knowall.socialstats.api.twitter;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+class ShowMeWhatYouGotApplication {
```

Comment:
We need to  think about name for our application and for modules. Not now - but we can have this thing in the back of our mids.

Reply:
Sure, we should... I think we should also reinstate our daily meetings (or weekly meetings :D - it would be a more appropriate name)